python/py-xdg-base-dirs/Portfile

```diff
@@ -28,3 +28,14 @@ python.versions         38 39 310 311
 
 python.pep517           yes
 python.pep517_backend   poetry
+
+if {${name} ne ${subport}} {
+    post-destroot {
+        set docdir ${prefix}/share/doc/${subport}
+        xinstall -d ${destroot}${docdir}
+        xinstall -m 0644 -W ${worksrcpath} README.md LICENCE \
```

Comment:
looks like LICENSE is misspelled or was that intentional...

Reply:
Yep, I’ve reported that to upstream: https://github.com/srstevenson/xdg-base-dirs/issues/180

Reply:
okay, also please tell them not to install that file into the `site-packages` directory as that is causing the issue here; that's not the correct place to put this type of files. Having that fixed is more important than correcting their spelling mistake ;)